website/addons/s3/tests/test_view.py

```diff
@@ -9,52 +9,35 @@
 from tests.base import OsfTestCase, get_default_metaschema
 from tests.factories import ProjectFactory, AuthUserFactory
 
+from website.addons.base import testing
+from website.addons.s3.provider import S3Provider
```

Comment:
I looks like `S3Provider`, `BUCKET_LOCATIONS` and `OsfTestCase` are not being used and can be removed